static/js/admin.js

```diff
@@ -6,7 +6,7 @@ const admin_settings_label = {
     realm_allow_edit_history: i18n.t("Enable message edit history"),
     realm_mandatory_topics: i18n.t("Require topics in stream messages"),
     realm_inline_image_preview: i18n.t("Show previews of uploaded and linked images"),
-    realm_inline_url_embed_preview: i18n.t("Show previews of linked websites"),
+    realm_inline_url_embed_preview: i18n.t(`Show previews of linked websites <a href="/help/allow-image-link-previews" target="_blank"><i class="fa fa-question-circle-o" aria-hidden="true"></i></a>`),
```

Comment:
You should break this into two lines.

Reply:
**Is this okay?**
```suggestion
realm_inline_url_embed_preview: 
    i18n.t(`Show previews of linked websites <a href="/help/allow-image-link-previews" target="_blank"><i class="fa fa-question-circle-o" aria-hidden="true"></i></a>`),
```
Breaking it further results in ESLinter saying `i18n string should not be a multiline string`.

If there is any workaround please let me know.

Reply:
@vinitS101 shouldn't running ``` ./tools/lint --fix ``` do the job?

Reply:
Do you mean to use backticks there?  Seems like it should be a use case for single-quote.  

In any case, I don't think we need to wrap the `<a>` element in `i18n.t`; it has no text in it.  

And actually the best approach might be to do a block like what we have for `push_notifications_tooltip` in `settings_checkbox.hbs`.
```
    {{#if push_notifications_tooltip}}                                                               
    <i class="fa fa-question-circle settings-info-icon" {{#if realm_push_notifications_enabled}}styl\
e="display:none"{{/if}}  data-toggle="tooltip"                                                       
      title="{{t 'Mobile push notifications are not configured on this server.' }}"></i>             
    {{/if}}                                                                                          
```